### PR TITLE
Fix formatting in burgers single stack tutorial

### PR DESCRIPTION
**Pull request by charleskawczynski**
# Description

Fix equation formatting in burgers single stack tutorial (adds line breaks to the equations).

I have

- [ ] Written and run all necessary tests with CLIMA by including `tests/runtests.jl`
- [ ] Followed all necessary [style guidelines](https://CliMA.github.io/CLIMA/latest/CodingConventions.html) and run `julia .dev/climaformat.jl .`
- [ ] Updated the documentation to reflect changes from this PR.

# For review by CLIMA developers

- [ ] There are no open pull requests for this already
- [ ] CLIMA developers with relevant expertise have been assigned to review this submission
- [ ] The code conforms to the [style guidelines](https://CliMA.github.io/CLIMA/latest/CodingConventions.html) and has consistent naming conventions. `julia .dev/format.jl` has been run in a separate commit.
- [ ] This code does what it is technically intended to do (all numerics make sense physically and/or computationally)

### CODE REVIEW

**Comment by ilopezgp** [comment]
Looks good to me. On another note, what is the standard format for vectors in the documentation? I did not write them in **bold**, but would like to make sure that equation (2) is understood as a vector equation.

File: tutorials/Atmos/burgers_single_stack.jl [hunk 1]
```diff
@@ -2,21 +2,25 @@
...
-# z_{\mathrm{min}}: ρcT = T=T_{\mathrm{fixed}}
-# z_{\mathrm{max}}: ρ = 1
-# z_{\mathrm{max}}: ρu = 0
-# z_{\mathrm{max}}: ρcT = \mathrm{no flux}
-# ``
+# ```math
+# \begin{align}
+# z_{\mathrm{min}}: & ρ = 1 \\
+# z_{\mathrm{min}}: & ρu = 0 \\
+# z_{\mathrm{min}}: & T = T_{\mathrm{fixed}}/(ρc) \\
```
Comment (ilopezgp) [inline tutorials/Atmos/burgers_single_stack.jl:18]: This is numerically correct in our case (\rho c = 1), but should we remove \rho c for units consistency?

Comment (charleskawczynski) [reply tutorials/Atmos/burgers_single_stack.jl:18]: Agreed. Is it alright if we write this as `ρcT = ρc T_{\mathrm{fixed}}`?

**Comment by charleskawczynski** [comment]
> Looks good to me. On another note, what is the standard format for vectors in the documentation? I did not write them in **bold**, but would like to make sure that equation (2) is understood as a vector equation.

I'm not sure if we have a standard format (bold/using vec), but I agree that it would be nice to make it clear somehow. I'm used to following bold-non-italics for vectors, which may be nice to avoid dealing with the vec plus the overbar.

**Comment by ilopezgp** [comment]
> > Looks good to me. On another note, what is the standard format for vectors in the documentation? I did not write them in **bold**, but would like to make sure that equation (2) is understood as a vector equation.
> 
> I'm not sure if we have a standard format (bold/using vec), but I agree that it would be nice to make it clear somehow. I'm used to following bold-non-italics for vectors, which may be nice to avoid dealing with the vec plus the overbar. Also, we'll need a bunch more changes to keep things consistent.

Yes, I definitely prefer **bold** instead of vector overbar (Thinking about EDMF documentation and lots of averages)

File: tutorials/Atmos/burgers_single_stack.jl [hunk 2]
```diff
@@ -4,28 +4,29 @@
...
 # ```math
 # \begin{align}
 # z_{\mathrm{min}}: & ρ = 1 \\
-# z_{\mathrm{min}}: & ρu = 0 \\
-# z_{\mathrm{min}}: & T = T_{\mathrm{fixed}}/(ρc) \\
+# z_{\mathrm{min}}: & ρ\mathbf{u} = 0 \\
+# z_{\mathrm{min}}: & ρcT = T_{\mathrm{fixed}}/(ρc) \\
 # z_{\mathrm{max}}: & ρ = 1 \\
-# z_{\mathrm{max}}: & ρu = 0 \\
+# z_{\mathrm{max}}: & ρ\mathbf{u} = 0 \\
```
Comment (ilopezgp) [inline tutorials/Atmos/burgers_single_stack.jl:20]: Thanksfor changing this. I think now we have to put the \rho c multiplying T_fixed on the RHS.

Comment (charleskawczynski) [reply tutorials/Atmos/burgers_single_stack.jl:20]: Yeah, I thought I had done that, updating now..

**Pull request merged**